mu4/scores/view/scorethumbnail.cpp

```diff
@@ -0,0 +1,25 @@
+#include "scorethumbnail.h"
+
+#include <QVariant>
+
+using namespace mu::scores;
+
+ScoreThumbnail::ScoreThumbnail(QQuickItem *parent) : QQuickPaintedItem(parent)
```

Comment:
The colon should be on the next line (https://github.com/musescore/Documentation/blob/master/CodeGuidelines.md)
It is best to set up autoformatting in QtCreator.
We use `uncrustify` with config file /tools/codestyle/uncrustify_musescore.cfg
or call manually

Reply:
Thank. Installed and configured uncrustify.